src/app/report/edit-display/edit-display.component.ts

```diff
@@ -103,7 +103,7 @@ export class EditDisplayComponent {
   }
 
   convertMessage(checkpoint: Checkpoint): string {
-    let message: string = checkpoint.message === null ? '' : checkpoint.message;
+    let message: string = checkpoint.message ?? '';
```

Comment:
But if message is an empty string, this whole function does nothing right?

Reply:
```suggestion
    let message: string = checkpoint.message;
```
I'll just change it to this for thread safety and leave it at that..

Reply:
Added the change.